lib/jekyll/entry_filter.rb

```diff
@@ -31,9 +31,9 @@ def relative_to_source(entry)
 
     def filter(entries)
       entries.reject do |e|
-        unless included?(e)
-          special?(e) || backup?(e) || excluded?(e) || symlink?(e)
-        end
+        next true if symlink?(e)
+        next false if included?(e)
```

Comment:
@parkr For knowledge purposes, can you pleas explain what these two lines mean..?

Thanks =)

Reply:
Added a comment!